### ISSUE TITLE
Mysql unrecognised service

### ISSUE THREAD

**Issue by MrVibe** [Varying-Vagrant-Vagrants/VVV#1043]
I am getting following error when trying to do vagrant up :  
`
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Checking if box 'ubuntu/trusty64' is up to date...
==> default: Machine already provisioned. Run `vagrant provision` or use the `--provision`
==> default: flag to force provisioning. Provisioners marked to run always will still run.
==> default: Running provisioner: shell...
    default: Running: inline script
==> default: mysql: unrecognized service
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
went wrong.
`

I have tried provisioning : vagrant up --provision and got the same error.
I have tried destroying the vagrant and VM completely and then reinstalled.
Using the Latest ubuntu image
I can login to the vagrant via SSH and have tried installing the mysql via gem but vagrant up always returns the same error.

Note that the Vagrant instance was working for past 6 months and did crash a few times but not like this. It would just not start. Any clues ?

**Comment by MrVibe** [comment]
Update : So, I logged in to my VM and installed the mysql service : **sudo apt-get install mysql-server**
but then it showed the error "**nginx: unrecognized service**" then I installed the nginx and then again I got the error : **bash: vagrant_up: command not found**

**Comment by MrVibe** [comment]
Nothing works ! All of a sudden everything stops working.

**Comment by jihgao** [comment]
+1

**Comment by jeremyfelt** [comment]
Hi @MrVibe, thanks for opening an issue. Can destroy the VM with `vagrant destroy` and then copy the contents of the next `vagrant up` into a gist (gist.github.com) or something similar so that we can see the entire provisioning process to look for errors?

**Comment by MrVibe** [comment]
Yes, forgot to update here, I was able to resolve this by following these steps, note that Vagrant halt/destroy commands were not working : 
1.  I removed the Vagrant virtual machine from the Virtual box manually from the GUI
2. Then renamed the vagrantfile for backup
3. Re-initialised the Vagrant by vagrant init 
4. Copied the backup file contents in the new vagrantfile 
5. Vagrant up 

now everything runs fine. Not sure exactly what happened.
